### PR TITLE
Static cards - turmoil

### PR DESCRIPTION
**Pull request by sienmich**

### CODE REVIEW

File: src/cards/turmoil/PoliticalAlliance.ts [hunk 1]
```diff
@@ -5,40 +5,44 @@ import {Player} from '../../Player';
...
-    public tags = [];
-    public name = CardName.POLITICAL_ALLIANCE;
-    public cardType = CardType.EVENT;
+export class PoliticalAlliance extends Card implements IProjectCard {
+  constructor() {
+    super({
+      cardType: CardType.EVENT,
+      name: CardName.POLITICAL_ALLIANCE,
+      cost: 4,
+      tags: [Tags.MICROBE, Tags.SCIENCE],
```
Comment (bafolts) [inline src/cards/turmoil/PoliticalAlliance.ts:19]: These tags are different. Original had no tags.

**Review by bafolts — CHANGES_REQUESTED** [review]
The tags on `PoliticalAlliance` changed from `[]` to `[Tags.MICROBE, Tags.SCIENCE]`? Was this expected?

File: src/cards/turmoil/PoliticalAlliance.ts (hunk 1 above)
Comment (sienmich) [reply src/cards/turmoil/PoliticalAlliance.ts:19]: It's probable some intelligent microbes spices plan to take control over the world. Thanks!

File: src/cards/turmoil/AerialLenses.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {PartyName} from '../../turmoil/parties/PartyName';
 import {RemoveAnyPlants} from '../../deferredActions/RemoveAnyPlants';
 import {CardRequirements} from '../CardRequirements';
 import {CardRenderer} from '../render/CardRenderer';
+import {Card} from '../Card';
```
Comment (sienmich) [inline src/cards/turmoil/AerialLenses.ts:11]:
```suggestion
```

**Comment by sienmich** [comment]
@bafolts, since you merged the same turmoil cards, here is now the rest of them.

I'll do today the colonies.

**Review by kberg — APPROVED** [review]

**Review by bafolts — APPROVED** [review]

**Pull request merged**